crates/bevy_ecs/src/query/iter.rs

```diff
@@ -69,6 +69,62 @@ where
             current_index: 0,
         }
     }
+
+    /// Consumes `self` and returns true if there were any elements remaining in this iterator.
+    #[inline(always)]
+    pub(crate) fn any_remaining(mut self) -> bool {
+        // NOTE: this mimics the behavior of `QueryIter::next()`, except that it
+        // never gets a `Self::Item`.
+        unsafe {
+            if self.is_dense {
+                loop {
+                    if self.current_index == self.current_len {
+                        let table_id = match self.table_id_iter.next() {
+                            Some(table_id) => table_id,
+                            None => return true,
```

Comment:
Pretty sure this isn't `any_remaining()`. It is `none_remaining`. Otherwise is_empty() would need to invert the result.

Reply:
Yep, that's correct :)

Reply:
(also a note to myself to update the comment)

Reply:
(also a note to myself to update the comment)

Reply:
@cart function name is fixed :)